modules/admin/admin-impl/src/main/java/com/enonic/xp/admin/impl/rest/resource/project/ProjectResource.java

```diff
@@ -60,7 +61,11 @@ public ProjectJson create( final MultipartForm form )
         throws Exception
     {
         final Project project = projectService.create( createParams( form ) );
-        return new ProjectJson( project );
+
+        final ProjectPermissions projectPermissions = getPermissionsFromForm( form );
+        projectService.modifyPermissions( project.getName(), getPermissionsFromForm( form ) );
```

Comment:
@vbradnitski you can use the `projectPermissions` variable instead of recall the `getPermissionsFromForm( form )` method.